MergerService/src/run.cs

```diff
@@ -270,24 +275,24 @@ private void RunTask(MergeTask task, ITaskUtils taskUtils, string? managerCallba
                         stopWatch.Reset();
                         stopWatch.Start();
 
-                        int totalTileCount = (int)bounds.Size();
+                        int singleTileBatchCount = (int)bounds.Size();
```

Comment:
check with @shimoncohen what is the size of tile in zoom 0 on zoom 21, is it long or int?